testsuite/integration/basic/pom.xml

```diff
@@ -1229,6 +1270,45 @@
                                 <phase>none</phase>
                             </execution>
 
+                            <!-- Tests against the install with microprofile-platform support -->
+                            <execution>
+                                <id>microprofile-platform-layers.surefire</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <jboss.install.dir>${basedir}/target/wildfly-microprofile-platform</jboss.install.dir>
+                                        <jboss.home>${project.build.directory}/wildfly-microprofile-platform</jboss.home>
+                                        <jboss.home.dir>${project.build.directory}/wildfly-microprofile-platform</jboss.home.dir>
+                                        <jbossas.dist>${project.build.directory}/wildfly-microprofile-platform</jbossas.dist>
+                                        <jboss.dist>${project.build.directory}/wildfly-microprofile-platform</jboss.dist>
+                                        <module.path>${project.build.directory}/wildfly-microprofile-platform/modules${path.separator}${basedir}/target/modules</module.path>
+                                    </systemPropertyVariables>
+                                    <environmentVariables>
+                                        <JBOSS_HOME>${project.build.directory}/wildfly-microprofile-platform</JBOSS_HOME>
+                                        <!-- dummy env property for MicroProfile Config testing -->
+                                        <MPCONFIG_TEST_ENV_VAR>integ-basicmpconfig-test-env-var-value</MPCONFIG_TEST_ENV_VAR>
+                                        <TEST_PROP>JMXPropertyEditorsTestCase-WFLY-12899</TEST_PROP>
+                                    </environmentVariables>
+                                    <redirectTestOutputToFile>${testLogToFile}</redirectTestOutputToFile>
+                                    <includes>
+                                        <include>org/wildfly/test/integration/microprofile/**/*TestCase.java</include>
+                                    </includes>
+                                    <excludes>
```

Comment:
@rhusar what drove the decision to exclude these tests? I am asking just for getting some understanding ... (I already asked on the AD, so if you answer there I'll copy the answer here and vice-versa)

Reply:
@tommaso-borgato These tests have other dependencies in order to run (e.g. security) while the profile only provisions base server (`jaxrs-server`) with the `microprofile-plaform` layer and of course `jmx-remoting` & `security-manager` which are needed by Arquillian. We might consider adding more layers to support more tests, but I am not sure if that doesn't defeat the purpose of the test.

Reply:
@rhusar no, that's fine .... it was just for understanding

Reply:
@rhusar .... this layer looks like one of the biggest candidate for adoption, also given we are in the XP 2 context; 

what about adding one or more galleon-maven-plugin & maven-surefire-plugin executions in order to run the excluded tests on wildfly servers taylored with the minimal number of necessary layers? Something analogous will be done for the JSF layer with regard to the EJB layer (@yersan is that correct?) ...

Reply:
@tommaso-borgato We can certainly do that, the price we pay is the extra testsuite execution time which is kind of exploding atm. Do you have a link handy where the JSF+EJB layer tests are discussed by any chance?

Reply:
@rhusar I am waiting for EAP7-1529 to be completed with JSF+EJB test, that's why I tagged @yersan: here is the TP snippet relative to this: https://gitlab.mw.lab.eng.bos.redhat.com/jbossqe-eap/test-plans/merge_requests/380/diffs#4e1bf40d542063d096fcd61d1fb6f49b4aba74ab_0_177

Probably he can contribute to this discussion ....

Reply:
@rhusar in addition to that, microprofile tests are fast (about 10 seconds on my laptop)

Reply:
^^^ do you have any update on the missing tests? would it be possible, for example, to add security tests on a slimmed server with the minimum amount of necessary layers?

Reply:
Actually, it looks like the problem is only with the test setup. I will put this on the backlog, but its not clear what those tests are missing – will need to check with Darran when he comes back.